docs/security/authentication/azure-ad-authentication.md

```diff
@@ -5,161 +5,151 @@ position: 1
 :::
 
-1. Login to the "[Azure Portal](https://portal.azure.com)" , click on your account positioned at the top-right of the screen, then select your desired directory.
+1. Log in to the [Azure Portal](https://portal.azure.com), click on your account positioned at the top-right of the screen, then select your desired directory.
 
-   ![Switch Azure Directories](images/switch-azure-directories.png "width=500")
+   ![Switch Azure Directories](images/aad-portal.png "width=500")
 
-2. Click the **All services** button and select the **Azure Active Directory** service.
+2. Select the hamburger menu in the [Azure Portal](https://portal.azure.com)  and select **Azure Active Directory** from the Azure menu.
 
    ![Open AzureAD service](images/aad-service.png "width=500")
 
 3. Select **App registrations** then choose **New application registration**
 
-   ![New App registration](images/new-app-registration.png "width=500")
+   ![New App registration](images/aad-new-app-registration.png "width=500")
 
-4. Choose a **Name** like *Octopus Deploy*, specify **Application Type** of *Web app/API*, and enter a value for **Sign-On URL** like *https://octopus.example.com* Then click **Create**.
- - The value you specify for Name will appear at the top of the Azure authentication page when the users are entering their credentials.
- - The value you specify for Sign-On URL should be the URL to your Octopus Server. This address is linked within your browser, so only has to be resolvable on your network, not from the public Internet.
+4. Choose a **Name** like *Octopus Deploy*, select the correct **Supported account type**  for Single or Multi-Tenant, and enter a value for **Sign-On URL** like *https://octopus.example.com/api/users/authenticatedToken/AzureAD*. Then click **Register**.
+ -The URL must use HTTPS
+ -When users input their credentials, the value you specify for Name will appear at the top of the Azure authentication page.
+ -The value you specify for Sign-On URL should be the URL to your Octopus Server. This address is linked within your browser, so it only has to be resolvable on your network, not from the public Internet.
+ -Include `/api/users/authenticatedToken/AzureAD` at the end of your Octopus URL
 
-   ![Filling the App registration form](images/new-app-registration-form.png "width=500")
+ :::hint
+ Please take care when adding this URL! They are **case-sensitive** and can be sensitive to trailing **slash** characters. You cannot use `HTTP` here, so you will need to use `https`. You will need to use an SSL certificate from a Certificate Authority, such as [LetsEncrypt](https://letsencrypt.org/) or [GoDaddy](https://godaddy.com) and you can also do this by using Octopus Deploy [Let's Encrypt Integration](/docs/security/exposing-octopus/lets-encrypt-integration.md) or one from Active Directory Certificate Services.
```

Comment:
Suggestion only, Let's Encrypt is a solid recommendation, I don't think we need to add others :)
```suggestion
 Please take care when adding this URL! They are **case-sensitive** and can be sensitive to trailing **slash** characters. You cannot use `HTTP` here, so you will need to use `https`. You will need to use an SSL certificate from a Certificate Authority, such as [LetsEncrypt](https://letsencrypt.org/) and you can also do this by using Octopus Deploy [Let's Encrypt Integration](/docs/security/exposing-octopus/lets-encrypt-integration.md) or one from Active Directory Certificate Services.
```